### PR TITLE
Plaid 2.0.0 jruby 1.7.22

### PR DESCRIPTION
**Pull request by betesh**
This passes on travis: https://travis-ci.org/betesh/plaid-ruby/builds/144196412

### CODE REVIEW

**Comment by betesh** [comment]
@benissimo please review

File: lib/plaid.rb [hunk 1]
```diff
@@ -51,12 +51,14 @@ def config
     #          as well.
     #
     # Returns a Hash with keys.to_sym (or nil if hash is nil).
-    def symbolize_hash(hash, values: false)
```
Comment (benissimo) [inline lib/plaid.rb]: Was `values` a hash then?

Comment (betesh) [inline lib/plaid.rb]: `values` is a keyword argument.  See https://robots.thoughtbot.com/ruby-2-keyword-arguments

Comment (benissimo) [inline lib/plaid.rb]: Yes, it was: "Note that the calling code is syntactically equal to calling a method with hash arguments, which makes for an easy transition from options hashes to keyword arguments."
https://robots.thoughtbot.com/ruby-2-keyword-arguments

👍

**Comment by benissimo** [comment]
This PR looks good @betesh 
Holding off on merging it til you give me the all clear.

**Comment by betesh** [comment]
Replaced by #2

**Pull request closed without merging**